src/Model/Filters.php

```diff
@@ -18,7 +18,7 @@
  * The version of the OpenAPI document: 1.0.0
```

Comment:
could you find possibility to make Filters, Filters1, ... with meaningful name, please?

Reply:
I'll take this later, coulldn't find a way to set a meaningful name